core/src/utilities/helpers/iframe-helpers.js

```diff
@@ -251,7 +252,7 @@ class IframeHelpersClass {
       : luigiDefaultSandboxRules;
 
     const iframe = document.createElement('iframe');
-    iframe.src = viewUrl;
+    iframe.src = ViewUrlDecorator.applyDecorators(viewUrl);
```

Comment:
Do we have to call this function if no `nodeViewURLDecorator` in settings.js is defined?

Reply:
I will add a method `hasDecorators()` to check if there are any decorators defined. Btw, its not tied to `nodeViewURLDecorate` setting, we can use this class from any place for other decorators too (like substitution, etc)